theme/package.json

```diff
@@ -8,16 +8,16 @@
     "scripts": {
         "dev": "webpack -d --env dev --config ./webpack.dev.js",
         "dev-watch": "watch 'npm run dev' src",
-        "live": "npm-run-all -p dev-watch proxy:wait",
+        "live": "npm-run-all -p dev-watch proxy browsersync",
         "proxy": "aem-site-theme-builder live",
-        "proxy:wait": "sleep 1 && npm run proxy",
         "prod": "webpack -p --config ./webpack.prod.js",
-        "deploy": "aem-site-theme-builder deploy"
+        "browsersync": "browser-sync start --proxy 'localhost:7000' --files 'dist'"
     },
     "devDependencies": {
         "@adobe/aem-site-theme-builder": "1.0.0",
         "autoprefixer": "^9.2.1",
         "bestzip": "^2.1.6",
+        "browser-sync": "^2.26.13",
```

Comment:
We could remove `^` in order to benefit from deterministic CI/CD results.

Reply:
Having `^` is recommended way of installing dependencies. It is done automatically when installing dependencies using `npm i --save-dev`. Moreover `^` can install only minor/patch updates and not major which should make CI/CD consistent. If we would like to keep dependencies without `^` then we should change all of current dependencies and compare them which `package-lock.json`.